src/pages/en/guides/troubleshooting.mdx

```diff
@@ -137,6 +137,12 @@ nodeLinker: "node-modules"
 
 In Astro, using a `<head>` tag works like any other HTML tag: it does not get moved to the top of the page or merged with the existing `<head>`. Because of this, you usually only want to include one `<head>` tag throughout a page. We recommend writing that single `<head>` and its contents in a [layout component](/en/core-concepts/layouts/).
 
+### An unexpected `<script>` or `<style>` is included
+
+You may notice an imported component's `<script>` or `<style>` tags included in your HTML source even if that component doesn't appear in the final output. For example, this will occur with [conditionally rendered](https://docs.astro.build/en/core-concepts/astro-components/#dynamic-html) components that are not displayed. 
```

Comment:
```suggestion
You may notice an imported component's `<script>` or `<style>` tags included in your HTML source even if that component doesn't appear in the final output. For example, this will occur with [conditionally rendered](/en/core-concepts/astro-components/#dynamic-html) components that are not displayed. 
```
pass link check